starter-code/index.html

```diff
@@ -2,30 +2,46 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel='stylesheet' href='css/main.css' style='text/css'>
 </head>
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+<div class='mainInfo'>
+<img class='phoneImg' src='/Users/shincero/code/labs/lab-css-instagram-clone/starter-code/images/phones.png'>
```

Comment:
All the images in the project break when opened from another pc that is not yours. Never use an absolute reference ( /Users/shincero/code/labs/lab-css-instagram-clone/starter-code/images/phones.png ) to the images, always opt for a relative reference  (./images/phones.png )